### PR TITLE
TAUR-1292 fall back to git rev-parse if no grok.sha file exists

### PR DESCRIPTION
**Pull request by jcasner**
@brev @unixorn please review

### CODE REVIEW

**Comment by brev** [comment]
:+1:

File: grok/grok/app/webservices/webapp.py [hunk 1]
```diff
@@ -99,7 +100,12 @@
 try:
   buildSha = open(os.path.join(grok.app.GROK_HOME, "static/grok.sha")).read()
 except IOError:
-  buildSha = '0'
+  try:
+    buildSha = check_output(["git", "rev-parse", "--verify", "HEAD"])
+  except CalledProcessError:
+    # Do nothing; just set to 0 as below
+    pass
+  buildSha = "0"
```
Comment (unixorn) [inline grok/grok/app/webservices/webapp.py]: I think this needs to be inside the except clause or we'll always reset to 0 even if git rev-parse returns a sha

Comment (jcasner) [inline grok/grok/app/webservices/webapp.py]: Good catch!

**Comment by jcasner** [comment]
@unixorn updated with your feedback

**Comment by unixorn** [comment]
:+1:

**Pull request merged**